### PR TITLE
Civ 1678 improve coverage

### PR DESCRIPTION
**Pull request by GorilaNaranja**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-1678

### Change description ###
Refactoring: improve coverage.

**Does this PR introduce a breaking change?** (check one with "x")

```
[x] Yes
[ ] No
```

### CODE REVIEW

**Review by TatsyMaslova — APPROVED** [review]
Good stuff!!!

**Review by taomoh — COMMENTED** [review]
Great to see how this reduces the number of code lines in tests. Most of my comments are around what seem to be typos in naming loggers

File: src/main/views/features/response/statementOfMeans/partner/partner-pension.njk [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      {% set errors = form.getErrors() %}
+      {% if errors %}
+        {{ govukErrorSummary({
+          "titleText": t("There was a problem"),
+          "errorList": errors
+        }) }}
+      {% endif %}
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+          <form method="post">
```
[the remark this answers is not in the document]
Comment (GorilaNaranja) [reply src/main/views/features/response/statementOfMeans/partner/partner-pension.njk:21]: I will tell Sergio to change it on his branch, thanks!

File: src/main/modules/statementOfMeans/partner/partnerPensionService.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+const {Logger} = require('@hmcts/nodejs-logging');
+const logger = Logger.getLogger('dependantsService');
+const partnerPension = new PartnerPension();
+
+export class PartnerPensionService {
+
+  public async getPartnerPension(claimId: string) {
+    try {
+      const case_data = await getCaseDataFromStore(claimId);
+      if (case_data && case_data.statementOfMeans && case_data.statementOfMeans.partnerPension) {
```
[the remark this answers is not in the document]
Comment (GorilaNaranja) [reply src/main/modules/statementOfMeans/partner/partnerPensionService.ts:15]: There is another issue to implement lodash in all the code. I will change it there!

**Pull request merged**